opentdf-with-hsm.yaml

```diff
@@ -11,6 +11,8 @@ logger:
 services:
   kas:
     enabled: true
+    issuer: http://localhost:8888/auth/realms/opentdf
```

Comment:
`issuer` is listed in the OIDC discovery URL. is this `issuer` intended as an override? If it is maybe we should clarify.

Reply:
good point.  the code should change to reflect that this is an override.  The initial reason is private and public URLs can exist for the same IdP